packages/editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -11,7 +11,14 @@ const FormatToolbar = ( { controls } ) => {
 				{ controls.map( ( format ) =>
 					<Slot name={ `RichText.ToolbarControls.${ format }` } key={ format } />
 				) }
-				<Slot name="RichText.ToolbarControls" />
+				<Slot name="RichText.ToolbarControls">
+					{ ( fills ) =>
+						<Toolbar
+							isCollapsed
+							controls={ fills.map( ( fill ) => fill[ 0 ].props ) }
```

Comment:
@youknowriad I think you originally added `isCollapsed` flag. Would it be okey to refactor `<DropdownMenu />` to take `children` as prop so we could pass all fills directly?